letsencrypt-vesta

```diff
@@ -254,8 +254,8 @@ do
         cp /etc/letsencrypt/live/$MAINDOMAIN/chain.pem $TMPLOC/$DOMAIN.ca
 
         #Check if the site already has a cert
-        HAS_CERT=`$VESTA_PATH/bin/v-get-web-domain-value $USER $DOMAIN SSL`
-        if [[ $HAS_CERT == 'no' ]]
+        HAS_CERT=`$VESTA_PATH/bin/v-list-web-domain-ssl $USER $DOMAIN`
+        if [[ $HAS_CERT == '' ]]
```

Comment:
There is a better way to detect empty strings in Linux operating systems.

Use this instead:
`if [[ -z $HAS_CERT ]]`
Using the previous, causes an error in Debian 8 where it does not recoginize '' as empty because there is a carrage return on the output of the `$VESTA_PATH/bin/v-list-web-domain-ssl $USER $DOMAIN` call.

For documentation purposes, it is noteworthy that the new v-list-web-domain-ssl script either returns nothing (when there is no certificate) or the details of the certificate itself.